__tests__/hooks/useLoanUnderwriter.test.ts

```diff
@@ -59,7 +59,7 @@ describe('useLoanUnderwriter', () => {
     });
   });
 
-  it('signals an error if you try to underwrite before having an account', async () => {
+  it('signals an error if you try to ugnderwrite before having an account', async () => {
```

Comment:
lol, is this intentional

Reply:
lol no, that was to trigger the test watcher, will update